io.github.listen1.Listen1.yml

```diff
@@ -0,0 +1,113 @@
+app-id: io.github.listen1.Listen1
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  # Node.js version: https://github.com/listen1/listen1_desktop/blob/7636b7d685e999a1894d393e64e3f8324280e6e5/.github/workflows/release.yml#L21
+  - org.freedesktop.Sdk.Extension.node16
+command: listen1
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-music
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.PowerManagement
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+modules:
+  - name: listen1
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: /usr/lib/sdk/node16/bin:/run/build/listen1/flatpak-node/chromedrive
+      env:
+        # Don't add ELECTRON_CACHE
+        XDG_CACHE_HOME: /run/build/listen1/flatpak-node/cache
+        npm_config_nodedir: /usr/lib/sdk/node16
+        npm_config_offline: "true"
+        npm_config_no_save: "true"
+        npm_config_cache: /run/build/listen1/flatpak-node/npm-cache
+        npm_config_loglevel: verbose
+        # https://stackoverflow.com/questions/25146976/can-i-get-npm-gyp-to-use-ccache
+        CXX: ccache g++
```

Comment:
I'm pretty sure this is not needed in flatpak-builder. the `--ccache` option will handle it